packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1263,7 +1296,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       this.setA11ySelectionMessage(getA11ySelectionMessage.onRemove(poppedItem))
     }
 
-    this.setStateAndInvokeHandler('onSelectedChange', null, { value })
+    this.setStateAndInvokeHandler(['onSelectedChange'], null, { value })
```

Comment:
`onChage` may be a better alternative than `onSelectedChange` is it already exists as a concept in the html `select` element